pom.xml

```diff
@@ -0,0 +1,244 @@
+						</includes>
+					</configuration>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-buildtimestamp-jgit</artifactId>
+					<version>${tycho.extras.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-eclipserun-plugin</artifactId>
+					<version>${tycho.extras.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-p2-extras-plugin</artifactId>
+					<version>${tycho.extras.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-source-feature-plugin</artifactId>
+					<version>${tycho.extras.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-source-plugin</artifactId>
+					<version>${tycho.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho</groupId>
+					<artifactId>tycho-p2-plugin</artifactId>
+					<version>${tycho.version}</version>
+				</plugin>
+				<plugin>
+					<groupId>org.eclipse.tycho.extras</groupId>
+					<artifactId>tycho-dependency-tools-plugin</artifactId>
+					<version>1.7.0</version>
+					<executions>
+						<execution>
+							<id>list-dependencies</id>
+							<goals>
+								<goal>list-dependencies</goal>
+							</goals>
+							<phase>compile</phase>
+						</execution>
+					</executions>
+				</plugin>
+			</plugins>
+		</pluginManagement>
+
+		<plugins>
+			<plugin>
+				<!-- This plugin configuration block is only needed if the repository contains plug-ins that don't have any sources. If it is omitted
+		 		Tycho will complain. -->
+				<groupId>org.eclipse.tycho.extras</groupId>
+				<artifactId>tycho-source-feature-plugin</artifactId>
+				<executions>
+					<execution>
+						<id>source-feature</id>
+						<phase>package</phase>
+						<goals>
+							<goal>source-feature</goal>
+						</goals>
+					</execution>
+				</executions>
+				<configuration>
+					<excludes>
+                        <plugin id="org.rdkit.knime.bin.linux.x86_64"/>
+                        <plugin id="org.rdkit.knime.bin.win32.x86_64"/>
+                        <plugin id="org.rdkit.knime.bin.macosx.x86_64"/>
+					</excludes>
+				</configuration>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-maven-plugin</artifactId>
+				<extensions>true</extensions>
+			</plugin>
+			<plugin>
+				<groupId>org.eclipse.tycho</groupId>
+				<artifactId>tycho-packaging-plugin</artifactId>
+				<dependencies>
+					<dependency>
+						<groupId>org.eclipse.tycho.extras</groupId>
+						<artifactId>tycho-buildtimestamp-jgit</artifactId>
+						<version>${tycho.extras.version}</version>
+					</dependency>
+				</dependencies>
+				<configuration>
+					<timestampProvider>jgit</timestampProvider>
+					<format>'v'yyyyMMddHHmm</format>
```

Comment:
As discussed, would it be possible to use instead of 'v' actually 'v${COMPANY_BUILD}', where COMPANY_BUILD is an environment variable that is sometimes defined, and sometimes not? When building it for NIBR, I would set that environment variable to "nibr", so that all NIBR internal build version numbers would look like "vnibr202006012211". That's how I do it today.

Reply:
I added a maven property `qualifier.prefix` that can be used to set this. You can use it like this:
`mvn clean verify -Dqualifier.prefix=vnibr`